<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodesDock</class>
 <widget class="QDockWidget" name="NodesDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nodes</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="m_search_nodes_edit">
        <property name="placeholderText">
         <string>Search nodes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="flow_document::NodesView" name="m_nodes_view">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_toolbar_layout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>flow_document::NodesView</class>
   <extends>QTreeView</extends>
   <header>flow_document/component/nodes/nodes_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/plugins/flow_document/flow_document.qrc"/>
 </resources>
 <connections/>
</ui>
